solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/ModelQuery.java

```diff
@@ -214,16 +214,15 @@ public ModelWeight(IndexSearcher searcher, FeatureWeight[] modelFeatures,
      * Goes through all the stored feature values, and calculates the normalized
      * values for all the features that will be used for scoring.
      */
-    public void normalize() {
+    public void makeRawFeatures() {
```

Comment:
This isn't so easy. It also needs modelFeatureValues for updating. The harder piece is it needs the actual featureWeight so it can get the default value. I don't think this would be an easy one